src/mprismetadata.cpp

```diff
@@ -19,6 +17,210 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+/*!
+    \qmltype MprisMetaData
+    \inqmlmodule Amber.Mpris
+    \brief Provides media metadata information
+
+    Provides access to information about media content.
+
+    All properties may be undefined if not applicable or known.
+    A default trackId will be returned if one is not defined.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::trackId
+    \brief Identifies the track inside a playlist
```

Comment:
This being DBus object path and the playlist support marked as not implemented, does it provide anything now? Should we also skip here, either making internal or removed?

Reply:
This value should be passed in as the argument for setPosition, and the call is only valid if the track id matches. So not having this kind-of prevents setPosition from working.

Reply:
Looks like the current jolla-mediaplayer PR doesn't set anything, and on the library level the trackId passed from remote party to SetPosition() call almost untouched to setPositionRequested() signal. So up to the app side whether it cares about the trackId? Though granted that if the trackId is passed in the request signal, it would be strange if it wasn't visible elsewhere on the API. And guess this is good even without playlist as it would allow to prevent seeks on a song that already changed.